example/javascriptlib/dependencies/2-unmanaged-packages/foo/src/foo.ts

```diff
@@ -0,0 +1,6 @@
+import {sortBy} from 'node_modules/lodash'
```

Comment:
Where does the identifier `node_modules/lodash` come from? Is it from the name of the `tgz` file? If so it should be explained in a bit more detail in the doccomments above